IPython/core/debugger.py

```diff
@@ -621,3 +621,13 @@ def do_where(self, arg):
                 self.print_stack_trace()
 
         do_w = do_where
+
+
+def set_trace(frame=None):
```

Comment:
Makes sense, though it takes a frame object, not an offset number. Here's the `set_trace` definition from pdb:

```python
def set_trace():
    Pdb().set_trace(sys._getframe().f_back)
```